crates/blockless-drivers/src/cgi_driver/process.rs

```diff
@@ -1,14 +1,44 @@
-use std::process::Stdio;
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+    process::Stdio,
+    sync::Once,
+};
 
 use crate::CgiErrorKind;
 use json::object::Object as JsonObject;
 use json::JsonValue;
-use log::debug;
+use log::{debug, error};
 use tokio::{
+    fs::File,
     io::{AsyncReadExt, AsyncWriteExt},
     process::{Child, Command},
 };
 
+const DB_NAME: &str = ".extsdb";
+
+use super::db::{ExtensionMeta, ExtensionMetaStatus, DB};
+
+fn get_db(path: impl AsRef<Path>) -> Option<&'static mut DB> {
+    static mut DB: Option<DB> = None;
+    static DB_ONCE: Once = Once::new();
+    DB_ONCE.call_once(|| unsafe {
+        let db = DB::new(path)
+            .map_err(|e| error!("error open db {}", e))
+            .ok();
+        DB = db
+            .map(|mut db| {
+                if db.create_schema().is_ok() {
+                    Some(db)
+                } else {
+                    None
+                }
+            })
+            .flatten();
```

Comment:
It looks like the error handling when creating the db instance could be improved. Instead of just ignoring the error, it might be more helpful to return a Result that can provide more information about what went wrong. This way, it would be easier to troubleshoot issues if they arise.
在创建db实例时，可能会有更好的错误处理方法。而不是简单地忽略错误，可以返回一个“Result”，它可以提供有关发生了什么问题的更多信息。这样，如果出现问题，就会更容易解决问题。